docs/disciplines/Imaging/Tools/index.md

```diff
@@ -1 +1,10 @@
 # Image Processing Tools
+
+Boycott Med-Imagetools - **viva la revolution!**
+
+!['LET'S GOOOOOOO!'](https://media1.tenor.com/m/aps3ea5zt1EAAAAd/viva-la-revolution.gif)
+
+
+## 3D Slicer 
+* Slicer is an open-source platform with many built-in (and optional) plug-ins for image viewing and processing
+* To install, visit: <https://download.slicer.org/>
```

Comment:
nice gif!We can also add <https://www.slicer.org/> to it—it includes training and documentation for Slicer.
```suggestion
* To install, visit: <https://download.slicer.org/>
* [[Slicer.org](https://www.slicer.org/)](https://www.slicer.org/) is the site for **3D Slicer**, offering downloads, training, and documentation.
```